sprokit/processes/core/handle_descriptor_request_process.cxx

```diff
@@ -128,50 +152,53 @@ ::_step()
 
   request = grab_from_port_using_trait( descriptor_request );
 
-  // Special case, output empty results
+  // Special case, output empty results and pass thru if not specified
   if( !request )
   {
     push_to_port_using_trait( track_descriptor_set, vital::track_descriptor_set_sptr() );
-
-    push_to_port_using_trait( image, vital::image_container_sptr() );
-    push_to_port_using_trait( timestamp, vital::timestamp() );
-    push_to_port_using_trait( filename, "" );
-    push_to_port_using_trait( stream_id, "" );
-    return;
+    return; // Normal return, no failure
   }
 
-  // Get output matrix and detections
+  // Get output descriptors from internal pipeline
   vital::track_descriptor_set_sptr descriptors;
-  std::vector< vital::image_container_sptr > images;
 
-  vital::string_t filename;
-  vital::string_t stream_id;
+  // Get filepaths
+  boost::filesystem::path p( request->data_location() );
```

Comment:
We are trying to remove boost usage in KWIVER. Is there an alternative using kwiver-sys to do this?